numpyro/infer/hmc_util.py

```diff
@@ -209,8 +209,8 @@ def update_fn(step_size, inverse_mass_matrix, state):
     return init_fn, update_fn
 
 
-def find_reasonable_step_size(potential_fn, kinetic_fn, momentum_generator, inverse_mass_matrix,
-                              position, rng_key, init_step_size):
+def find_reasonable_step_size(potential_fn, kinetic_fn, momentum_generator,
+                              init_step_size, inverse_mass_matrix, position, rng_key):
```

Comment:
I just reorder the position of `init_step_size` to clean up the code (using the utility `identity`).